test_copilot_api.py

```
@@ -0,0 +1,225 @@
+        print(f"   - Status Text: {response.reason}")
+        
+        # Check if request was successful
+        if not response.ok:
+            print(f"❌ HTTP Error Details:")
+            print(f"   - Status Code: {response.status_code}")
+            print(f"   - Status Text: {response.reason}")
+            
+            try:
+                error_response = response.json()
+                print(f"📄 Response Body: {json.dumps(error_response, indent=2)}")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)